### ISSUE TITLE
Bug: Telescope plugin configuration does not work

### ISSUE THREAD

**Issue by chrisgrieser** [debugloop/telescope-undo.nvim#1]
Thanks for the plugin, undotree was one of the few non-lua plugins I still have.

I tried configuring the extension's  telescope layout, but it seems that these configs have no effect?

```lua
require("telescope").setup {
        extensions = {
		undo = {
			prompt_prefix = "?",
			initial_mode = "normal",
		},
	},
}
```

**Comment by debugloop** [comment]
Yup, there is no configuration right now, I really meant it in the Readme ;) I hope to look into it tomorrow.

**Comment by debugloop** [comment]
I looked into it but I'm not sure how to accomplish that, or where I am dropping some opts that should get applied... Can you name an extension that can -- for instance -- have its prompt overwritten as in your example above? I'm by no means an advanced telescope user and haven't configured extensions myself yet, but it seems for me that the extensions section is meant for specific settings, not overwrites of telescope-wide opts?

**Comment by chrisgrieser** [comment]
Hmm, not too sure, I don't use many telescope extensions myself, this one is one of my firsts 😅

maybe have a look at https://github.com/nvim-telescope/telescope.nvim/wiki/Extensions ?

**Comment by debugloop** [comment]
I've implemented some basic configuration (`use_delta`, see README) and I'm pretty sure that the `extensions` section of telescope's setup is not the place for extension-specific overrides, but for settings implemented by an extension. If you find that this extension doesn't respect telescope-wide settings please reopen with an example 👍🏻
